gateway/utils/handlers/config_crud.go

```diff
@@ -560,6 +559,39 @@ func HandleInspectCollectionSchema(adminMan *admin.Manager, modules *modules.Mod
 	}
 }
 
+// HandleUntrackCollectionSchema gets the schema for particular collection & removed the collection from the database collection schema in config
```

Comment:
HandleUntrackCollectionSchema removes the collection from the database config 
```suggestion
// HandleUntrackCollectionSchema gets the schema for particular collection & removed the collection from the database collection schema in config
```